libsplinter/src/biome/rest_api/config.rs

```diff
@@ -28,6 +29,8 @@ pub struct BiomeRestConfig {
     issuer: String,
     /// Duration of JWT tokens issued by this service
     access_token_duration: Duration,
+
+    refresh_token_duration: Duration,
```

Comment:
For consistency with the other fields, please add a comment about this field.

Reply:
updated